project-overview.md

```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
 ### Tokens
 
 Many real world attestation solutions and potential extrapolations of their use have been considered to ensure that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
```

Comment:
```suggestion
To ensure that the code backs up the design, reference support for multiple attestation evidence formats are supported in the project. The following are currently supported:
```

Reply:
The title is tokens for the section and we have used attestation tokens so I would prefer to retain this terminology!

Reply:
as pointed out by Sergei in another review comment, some of these (namely DICE) are not strictly "tokens"

Reply:
This has been addressed in the titles and clarified on the individual specific bullets